pallets/schema/src/tests.rs

```diff
@@ -314,3 +314,30 @@ fn test_schema_lookup() {
 		}
 	});
 }
+
+#[test]
+fn test_max_encoded_schema_limit_exceeded_create() {
+	// Arrange
+	let mut mock_storage = MockStorage::new();
+
+	// Act
+	let large_schema_data = vec![0u8; <Test as frame_system::Config>::MaxEncodedSchemaLength + 1];
```

Comment:
@HarishKarthickS I don't think this can be done, as the input is compiled to be of type `BoundedVec` and not just `Vec`, where `MaxEncodedSchemaLength + 1` is impossible due to below,
`pub type InputSchemaOf<T> = BoundedVec<u8, <T as Config>::MaxEncodedSchemaLength>;
`
but it is possible for `MaxEncodedSchemaLength - 1` to take place with a `BoundedVec` since it will be below threshold input set limit.